pkg/executor/distsql.go

```diff
@@ -676,7 +676,6 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, workCh chan<
 			SetStartTS(e.startTS).
 			SetDesc(e.desc).
 			SetKeepOrder(e.keepOrder).
-			SetPaging(e.indexPaging).
```

Comment:
Because this line is useless, it will be overridden by the following `SetFromSessionVars`
https://github.com/pingcap/tidb/blob/249398128981fb1d5b9e943e2620c5628d488bf4/pkg/distsql/request_builder.go#L305